clang-tools-extra/docs/clang-tidy/ExternalClang-TidyExamples.rst

```diff
@@ -0,0 +1,32 @@
+=======================
+External Clang-Tidy Examples
+=======================
+
+Introduction
+============
+
+This page provides examples of what people have done with `clang-tidy` that 
+might serve as useful guides (or starting points) to develop your own checks. 
+They may be helpful for necessary things such as how to write CMakeLists.txt 
+for an out-of-tree plugin of `clang-tidy` checks.
+
+If you know of (or wrote!) a tool or project using clang-tidy, please share it 
```

Comment:
```suggestion
If you know of (or wrote!) a tool or project using Clang-tidy, please share it 
```

Reply:
That one is directly quoted from the repo's README.md, as https://clang.llvm.org/docs/ExternalClangExamples.html also directly quotes each repo's README.md. Is it fine for us to alter that?